section3/exercises/hashes.rb

```diff
@@ -1,54 +1,59 @@
 # In the below exercises, write code that achieves
 # the desired result. To check your work, run this
-# file by entering the following command in your terminal: 
+# file by entering the following command in your terminal:
 # `ruby section3/exercises/hashes.rb`
 
 # Example: Write code that prints a hash holding grocery store inventory:
-foods = {apples: 23, grapes: 507, eggs: 48}
-p foods
+# vfoods = {apples: 23, grapes: 507, eggs: 48}
+# p foods
 
 # Write code that prints a hash holding zoo animal inventory:
-zoo = #YOUR CODE HERE
+zoo = {penguins: 4, lions: 3, orangutans: 5} # usings symbols with : notation
 p zoo
 
-# Write code that prints all of the 'keys' of the zoo variable 
+# Write code that prints all of the 'keys' of the zoo variable
 # you created above:
-# YOUR CODE HERE
+puts zoo.keys
 
-# Write code that prints all of the 'values' of the zoo variable 
+# Write code that prints all of the 'values' of the zoo variable
 # you created above:
-# YOUR CODE HERE
+puts zoo.values
 
-# Write code that prints the value of the first animal of the zoo variable 
+# Write code that prints the value of the first animal of the zoo variable
 # you created above:
-# YOUR CODE HERE
+puts zoo.keys[0]
```

Comment:
I think you're actually looking for `puts zoo.values[0]` here or alternatively, `puts zoo[zoo.keys[0]]`.